--- app/frontend/genproto/farm_ng_proto/tractor/v1/resource.ts	2020-09-24 10:33:12.000000000 -0700
+++ protos/resource.patch.ts	2020-09-24 10:33:03.000000000 -0700
@@ -22,7 +22,7 @@
   /**
    *  (Optional) The uncompressed size of the resource in bytes.
    */
-  length: number | undefined;
+  length: Long | undefined;
 }

 export interface File {
@@ -127,7 +127,7 @@
       message.contentType = "";
     }
     if (object.length !== undefined && object.length !== null) {
-      message.length = Number(object.length);
+      message.length = Long.fromNumber(object.length);
     } else {
       message.length = undefined;
     }
@@ -146,7 +146,7 @@
       message.contentType = "";
     }
     if (object.length !== undefined && object.length !== null) {
-      message.length = object.length;
+      message.length = object.length as Long;
     } else {
       message.length = undefined;
     }
