--- resource.ts.2.orig	2020-10-12 12:30:16.052442002 -0700
+++ resource.ts	2020-10-12 12:31:20.459182709 -0700
@@ -22,7 +22,7 @@
   /**
    *  (Optional) The uncompressed size of the resource in bytes.
    */
-  length: number | undefined;
+  length: Long | undefined;
 }

 export interface File {
@@ -127,7 +127,7 @@
       message.contentType = "";
     }
     if (object.length !== undefined && object.length !== null) {
-      message.length = Number(object.length);
+      message.length = Long.fromNumber(object.length);
     } else {
       message.length = undefined;
     }
@@ -146,7 +146,7 @@
       message.contentType = "";
     }
     if (object.length !== undefined && object.length !== null) {
-      message.length = object.length;
+      message.length = object.length as Long;
     } else {
       message.length = undefined;
     }
